src/style.css

```diff
@@ -1 +1,35 @@
-/* add styles here */
+body {
+  margin: 0;
+
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+
+  padding: 2px;
+}
+
+.stars__star {
+  width: 13px;
+  height: 12px;
+  background-image: url('./images/star.svg');
+  background-size: contain;
+  background-repeat: no-repeat;
+
+  margin: 0;
+  padding: 0;
+  line-height: 1;
+
+  &:not(:last-child) {
+    margin-right: 7px;
+  }
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url('./images/star-active.svg');
+}
```

Comment:
The task requires using a container modifier and a pseudo-selector for the yellow stars, without extra classes. Consider revising this implementation to align with the task requirements.